dist/src/main/java/io/camunda/application/commons/backup/BackupComponent.java

```diff
@@ -19,11 +19,13 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
 import org.springframework.stereotype.Component;
 
 @Component
 @Configuration
+@Profile("operate | tasklist")
 public class BackupComponent {
```

Comment:
:thought_balloon: A little confusing, since it's more like `HistoryBackupComponent` or something :shrug:

Reply:
I have also changed the annotation in my local branch already